test/e2e/specs/infrastructure/infrastructure__ssr-enabled.ts

```diff
@@ -1,27 +1,26 @@
 /**
  * @group calypso-pr
- * @group kpi
```

Comment:
question (non-blocking): sorry, I missed this! Do we also need to remove this `kpi` group?

Reply:
Yep, this was left over from half a year ago when I was working on a dashboard, so it's actually been an unintentional change all along.